src/test/java/org/mitre/synthea/modules/DeathModuleTest.java

```diff
@@ -44,8 +46,9 @@ public void setup() throws IOException {
 
     long birthTime = time - Utilities.convertTime("years", 35);
     person.attributes.put(Person.BIRTHDATE, birthTime);
-    Payer.loadNoInsurance();
-    person.coverage.setPayerAtTime(time, Payer.noInsurance);
+    PayerManager.loadPayers(new Location("Massachusetts", null));
```

Comment:
Ideally, you do not want to specify a location in the unit tests like "Massachusetts" because it will break synthea-international support. Instead, you want to do something like

```
String testStateDefault = Config.get("test_state.default", "Massachusetts");
PayerManager.loadPayers(new Location(testStateDefault, null));
```